.github/workflows/links.yml

```
@@ -0,0 +1,24 @@
+name: Links
+
+on:
+  push:
+  pull_request:
+  repository_dispatch:
+  workflow_dispatch:
+  schedule:
+    - cron: "00 18 * * *"
+
+jobs:
+  linkChecker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Link Checker
+        uses: lycheeverse/lychee-action@v1.0.9
+        with:
+          args: --verbose --no-progress --require-https **/*.md **/*.html
```

```suggestion
          args: --verbose --no-progress **/*.md **/*.html
```

The argument is not recognized for some reason, so let's drop it.